backend/Annoucements.py

```diff
@@ -1,11 +1,15 @@
 import ids
 import time
 import Slack
+import requests
+import json
 
 myfirebase = ids.myfirebase
+FCM_KEY = 'AAAAfPkLsFE:APA91bH7vQ6_InhI1tBrr_k3tFCm7w3SvlU8Ba0FyrXE7GtOJN7dZEKctdW9Ota8mneIDeJA5Ddambw-ugyjbMZPyV9QkPtNft9z2FtIQ7WG_Ql7OHgYgnZ3dc8f-z334rTR8wjNkKDi'
```

Comment:
Aaaahhhh! 😱Force push and remove this!

Reply:
lol but now it is in the comments on github

Reply:
As long as you changed the api key it's fine